rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -749,42 +749,67 @@ public JobId submit(URL job, Map<String, String> variables, Map<String, String>
     @Override
     public JobId submit(URL job)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
-        return this.submit(job, Collections.<String, String> emptyMap(), Collections.<String, String> emptyMap());
+        return this.submit(job, null, null);
     }
 
     @Override
-    public JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName)
+    public JobId submit(URL job, Map<String, String> variables)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
-        return this.submitFromCatalog(catalogRestURL,
-                                      bucketName,
-                                      workflowName,
-                                      Collections.<String, String> emptyMap());
+        return this.submit(job, variables, null);
+    }
 
+    @Override
+    public JobId submit(Map<String, String> genericInfos, URL job, Map<String, String> variables)
+            throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
+        return this.submit(job, variables, genericInfos, null);
     }
 
     @Override
-    public JobId submit(File job, Map<String, String> variables)
+    public JobId submit(URL job, Map<String, String> variables, Map<String, String> requestHeaderParams)
+            throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)